avocado/core/remote/runner.py

```diff
@@ -270,6 +279,9 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
                 raise exceptions.JobError(details)
             sys.stdout = stdout_backup
             sys.stderr = stderr_backup
+        # returning the remote and local avocado version to be added to job.log as a possible warning
+        summary.add(self.remote_version.split('.')[0])
+        summary.add(self.local_version.split('.')[0])
```

Comment:
Let's not abuse `summary`. It's intended to contain types of test failures. Also, no need for the job to know about the version mismatch. At this point you can send the warning directly to the log.